challenges-webapp/src/main/webapp/vue-app/engagement-center/main.js

```diff
@@ -47,6 +47,9 @@ export function init(isAdministrator) {
   exoi18n.loadLanguageAsync(lang, urls).then(i18n => {
     // init Vue app when locale ressources are ready
     Vue.createApp({
+      mounted() {
+        document.dispatchEvent(new CustomEvent('hideTopBarLoading'));
+      },
```

Comment:
```suggestion
```

It's better to delete the first dispatch Event defined in line 33 to not have an infinite loading and then call `this.$root.$applicationLoaded()` when App Center is displayed like you can see in : https://github.com/Meeds-io/gamification/blob/3a95bcb435ff6e1b94f0534bbbb90acce538efc1/portlets/src/main/webapp/vue-app/popularSpaces/components/PopularSpaces.vue#L89 , and https://github.com/Meeds-io/social/blob/883748baa1396f3f6e9859a244d053075e3997c8/webapp/portlet/src/main/webapp/vue-apps/external-spaces-list/components/ExternalSpacesList.vue#L59
This call is made when the application is completely loaded the first time

Reply:
done !